src/templater.sh

```diff
@@ -48,6 +48,7 @@ function templater() {
 
   # cat "${templateDst}"
 
+  # shellcheck disable=SC2016
   if grep '${' "${templateDst}" && [ "${ALLOW_REMAINDERS}" = "0" ]; then
```

Comment:
Just added you as a collaborator 🎉 maybe that helps? I restarted the job and it's now way past the point of error already. Seems like apt sources is in BETA and maybe half of Travis' fleet has a broken version and half still works. Or something similar